experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -190,6 +190,15 @@ export interface TreeField extends Tree<FieldSchema>, Iterable<TreeNode> {
  * @alpha
  */
 export interface MapNode<TSchema extends MapSchema> extends TreeNode {
+	/**
+	 * Get the value associated with `key`.
+	 * @param key - which map entry to look up.
+	 *
```

Comment:
Just a suggestion (feel free to ignore, or postpone). If these methods are end-user facing, it might be nice to offer `@example`s outlining the differences between boxed and unboxed gets here.

Reply:
That's a great suggestion, though I will postpone, lest we waste time making examples that need to change along with the API during final revisions. We'll definitely be doing a "final docs pass" when the API is fully checked in and settled, and I'd love to use that time to write examples with you for as many things as we can. I really like examples in doc comments.